frontend/src/lib/forms/Select.svelte

```diff
@@ -4,7 +4,7 @@
   import FormField from './FormField.svelte';
   import { randomFormId } from './utils';
 
-  interface Props {
+  export interface Props {
```

Comment:
For next time, I would give/export these props with a more specific name, so that importers don't have to rename it themselves.

Reply:
Done in commit be1ef144d